documents/toolbox-population.md

```diff
@@ -0,0 +1,80 @@
+| \tools\netcoreapp31\VisualStudioToolsManifest.xml | .NET Core >= 3.1                                            |
+| \tools\VisualStudioToolsManifest.xml              | .NET Framework < 4.7, .NET Core < 3.1, and other frameworks |
+
+## Toolbox items from unreferenced NuGet packages (Visual Studio 16.7 Preview 2 and later)
+
+If a NuGet package in a NuGet fallback folder contains a [tools\VisualStudioToolsManifest.xml file](https://docs.microsoft.com/en-us/nuget/guides/create-ui-controls), Toolbox will show the Toolbox items listed in the manifest for any project that is compatible with the package, even projects that do not reference that package.
+
+### Limitations of the fallback folder approach
+
+* It currently only supports WPF .NET Core packages and projects.
+* It only supports one manifest per package, in the root of the tools directory. [Manifests for specific Target Framework Monikers](#target-framework-specific-manifests) are not supported.
+
+### Known issues in 16.7 Preview 2
+
+* Custom icons are not supported yet.
+* Toolbox item names are indented.
+* After a package is referenced, Toolbox shows two copies of the Toolbox items.
+
+### Getting started
+
+1. Create a NuGet package containing WPF .NET Core controls and a [tools\VisualStudioToolsManifest.xml file](https://docs.microsoft.com/en-us/nuget/guides/create-ui-controls). You could create your own package, or [build our sample package](#how-to-build-our-wpf-net-core-sample-package).
+2. Create a new directory somewhere on disk to serve as your NuGet fallback folder.
+3. Create a new text file with the extension .config (ex. MyFallbackFolder.config) in %ProgramFiles(x86)%\NuGet\Config.
+4. Add the following XML to the .config file, replacing c:\MyFallbackFolder with the actual path to your fallback folder.
+
+    ```xml
+    <?xml version="1.0" encoding="utf-8"?>
+    <configuration>
+      <fallbackPackageFolders>
+        <add key="FallbackFolderTest" value="c:\MyFallbackFolder" />
+      </fallbackPackageFolders>
+    </configuration>
+    ```
+
+5. Download nuget.exe as described [here](https://docs.microsoft.com/en-us/nuget/reference/nuget-exe-cli-reference).
+6. Expand your NuGet package into your fallback folder using the following command:
+
+    ```bat
+    nuget.exe add <path to your nupkg> -Source <path to your fallback folder> -Expand
+    ```
+
+7. Launch Visual Studio 16.7 Preview 2 or later.
+8. Create a new WPF .NET Core project.
+9. Open the Toolbox pane
+10. Wait for both the Toolbox and XAML designer to finish initializing.
+11. The Toolbox items defined in tools\VisualStudioToolsManifest.xml should appear in the Toolbox. If you're using our sample project, you should see a CustomControlLibrary.WpfCore tab in Toolbox containing a Toolbox item for the CustomButton control.
+12. Double-click one of the Toolbox items to add a control of that type to the XAML file.
```

Comment:
12. ... and add a package reference for the NuGet package to the project.

Reply:
Added your proposed text